modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -108,9 +130,14 @@ If you remove the `devworkspace-webhook-server` deployment without removing the
 +
 . Run the following commands to remove any lingering services, secrets, and config maps:
```

Comment:
I wouldn't feel comfortable using the word "lingering". I would use "remaining", if that is technically accurate.

Reply:
@sounix000 +1

Reply:
Replaced with `remaining`